spec/index.bs

```diff
@@ -1242,7 +1242,7 @@ that returns a new buffer in the [=buffer state/mapped=] or [=buffer state/unmap
 [Serializable]
 interface GPUBuffer {
     Promise<void> mapAsync(GPUMapModeFlags mode, optional GPUSize64 offset = 0, optional GPUSize64 size = 0);
-    ArrayBuffer getMappedRange(optional GPUSize64 offset = 0, optional GPUSize64 size = 0);
+    ArrayBuffer getMappedRange(optional GPUSize64 offset, optional GPUSize64 size = 0);
```

Comment:
(non-blocking) Like in https://github.com/gpuweb/gpuweb/pull/903#issuecomment-654585369, this one could also be just two overloads instead of three: getMappedRange(), getMappedRange(offset, size)